### ISSUE TITLE
Images - Add/improve alt text, link title, components

### ISSUE THREAD

**Issue by mheppler** [IQSS/dataverse#6106]
Some basic accessibility audits (see screenshot below of Audit panel in Safari's Web Inspector) of the app have pointed out a few places where alternate text or `alt` attributes are missing from `img` elements.

![Screen Shot 2019-08-20 at 2 36 20 PM](https://user-images.githubusercontent.com/687227/63374478-e6026b00-c357-11e9-8eeb-4d4d3df783e7.png)

There are roughly 30 images used in our app, most missing the `alt` attribute or do not have suitably descriptive text alternative (see: https://accessibility.huit.harvard.edu/describe-content-images).

Also, in regards to performance and optimizing page loading, there are a variety of UI components used to deliver images to the page (`h:graphicImage`: 12, `img`: 15, `p:graphicImage`: 4). Those components should be reviewed to determine if PrimeFaces or JSF components are necessary, compared to plain HTML `img` tags.

Additional note, these alt text values should be formatted using the bundle text in a way that will allow for easy translation as part of the internationalization efforts. (e.g. `alt="logotipo de Harvard Dataverse"` vs `alt="Harvard Dataverse logo"`.)

**Comment by djbrooke** [comment]
- We should put these in the bundle wherever possible so that they can be translated

**Comment by mheppler** [comment]
- maintenance.xhtml
    - [ ] add "logo" to alt text for Harvard Dataverse logo img in navbar
    - [ ] add "logo" to alt text for The Dataverse Project logo img in footer 

- dataset-license-terms.xhtml
    - [ ] add "Creative Commons CC0 1.0 Public Domain Dedication icon" alt text to CC0 license img in Terms tab of dataset pg

- dataset-widgets.xhtml
    - [ ] add dynamic {file name} alt text to Select File Thumbnail popup
    - [ ] add "Thumbnail image selected for dataset" alt text to image on Thumbnail form on dataset Thumbnail + Widgets pg

- dataset.xhtml
    - [ ] add dynamic {dataset title} alt text to thumbnail next to title on dataset pg

- dataverse.xhtml
    - [ ] add "logo" to alt text of dataverse logo in Feature Dataverses carousel on dataverse pg
    - [ ] REMOVED REFACTORED

- dataverse_footer.xhtml
    - [ ] add "logo" to alt text for The Dataverse Project logo img in footer, and **BUNDLE**

- dataverse_header.xhtml
    - [ ] DONE... NEEDS INTERNATIONALIZATION
    - [ ] add "logo" to alt text for rectangle dataverse theme logo img in header
    - [ ] add "logo" to alt text for square dataverse theme logo img in header

- dataverse_template.xhtml
    - [ ] move "Loading" alt text for blue (ajax) spinner to bundle

- dataverseuser.xhtml
    - [ ] add dynamic {persistentIdName} + "logo" alt text for auth provider on Account pg (view)
    - [ ] add dynamic {persistentIdName} + "logo" alt text for auth provider on Account pg (edit)

- editFilesFragment.xhtml
    - [ ] add dynamic {file name} alt text to file thumbnail img in Edit Files pg
    - [ ] add dynamic {file name} alt text to file thumbnail in Edit Files pg

- file-download-button-fragment.xhtml
    - [ ] LOGO REMOVED IN ANOTHER PR

- file-info-fragment.xhtml
    - [ ] add dynamic {file name} alt text to file thumbnail in ??File tab??

- file.xhtml
    - [ ] add dynamic {file name} alt text to file preview on file pg

- filesFragment.xhtml
    - [ ] add dynamic {file name} alt text to file thumbnail img in Files tab of dataset pg
    - [ ] add dynamic {file name} alt text to ??PREVIEW?? img in Files tab of dataset pg

- iframe.xhtml
    - [ ] move "Loading" alt text for blue (ajax) spinner to bundle

- loginpage.xhtml
    - [ ] add "ORCID" + "logo" alt text for ORICD auth provider on login pg

- /oauth2/firstLogin.xhtml
    - [ ] add dynamic {persistentIdName} + "logo" alt text for auth provider on (first login) Account pg

- search-include-fragment.xhtml
    - [ ] add dynamic {dataverse name} + "logo" alt text for dataverse result cards on Dataverse pg
    - [ ] add dynamic {dataset name} alt text for dataset result cards on Dataverse pg
    - [ ] add dynamic {file name} alt text for file result cards on Dataverse pg

- themeAndWidgetsFragment.xhtml
    - [ ] add dynamic {dataverse name} + "logo" alt text for existing dataverse theme logo img
    - [ ] add dynamic {dataverse name} + "logo" alt text for new upload dataverse theme logo img

**Comment by TaniaSchlatter** [comment]
Looks good! Minor question. I wonder about differentiating the Header logo from the Footer logo, with something like "Installation logo" and "Software creator logo"

**Comment by mheppler** [comment]
@TaniaSchlatter I think those suggested terms mean more to people who work on the Dataverse app and less to the people who use the app. If there is a demand to clarify these differences in the UI, maybe we review the "Powered by" text in the footer, as opposed to hiding these distinctions in the alt text of an image.

**Comment by mheppler** [comment]
Here is an example code snippet of the new (to us) OmniFaces method for combining dynamic content and bundle text in the alt text for an image.

```
<img src="/logos/#{dv.logoOwnerId}/#{dv.dataverseTheme.logo}" alt="#{of:format1(bundle['alt.logo'], dv.name)}" .../>
```
